pom.xml

```diff
@@ -165,21 +165,21 @@
     <google.guava.version>31.1-jre</google.guava.version>
     <hamcrest.version>2.2</hamcrest.version>
     <htmlunit.version>4.8.0</htmlunit.version>
-    <httpclient.version>4.5.14</httpclient.version>
+    <httpclient5.version>5.2.1</httpclient5.version>
```

Comment:
I had to update several major dependencies for the build to work with java 17 and the latest spring boot version. most notably:
- javax -> jakarta, of course
- jetty
- jaxb